### DISCUSSION TITLE
How are admin level 6 boundaries missing from OSM rendered in geoBoundaries ADM2?

### DISCUSSION THREAD

**Discussion by ksmithNau** [wmgeolab/geoBoundaries#3432] [Q&A]
I would like to know how you rendered the complete looking image of OSM ADM2 and what the source of this is. 

OSM-Boundaries dataset does not use ADM0-2 and OSM source appears to be missing entire counties at admin level 6 for many countries (Japan, Saudi Arabia, Thailand, Turkey, etc.). I understand that the ADM2 and OSM Admin Level 6 aren't a one-to-one comparison, which drives my question of how you're rendering ADM2 from OSM.

How are you rendering polygons that are missing from the underlying datasets? How can I query/extract what is being rendered?

**geoBoundaries UI for Japan OSM-Boundaries ADM2:**
![image](https://github.com/wmgeolab/geoBoundaries/assets/129110851/adb9763e-a9dc-49f2-87ea-bb295ec12c4c)

**OSM-Boundaries UI for Japan Admin 5 & 6:**
![image](https://github.com/wmgeolab/geoBoundaries/assets/129110851/d4169fa2-60c3-4433-849e-473b62d65ea1)

**OSM Raw Extract for Japan Admin Level 6:**
![image](https://github.com/wmgeolab/geoBoundaries/assets/129110851/d56eec16-0a14-42cf-9648-ef7e617c158d)

In my workflow, I initially thought this was an [issue](https://github.com/OvertureMaps/data/issues/106) with Overture Maps dataset, but discovered the upstream issue is in OSM.

**Comment by DanRunfola** [comment]
One of our former (graduated) Ph.D. students - Dr. Bahgat - was the one that worked on those pipelines, which can be found (I think) here:
https://github.com/wmgeolab/geoContrast

@karimbahgat any information you can provide on this one?  I'm not sure the exact crosswalk you used for OSM.

**Comment by ksmithNau** [comment]
@DanRunfola thanks for the pointer.

Sort of unrelated to this discussion topic, but something I realized in my workflow comparing OSM admin level 6 and geoBoundaries ADM2, the global ADM2 polygons have significantly lower vertex count than the individual country polygons. Is that what was meant by `extensive simplification` for the CGAZ downloads? If I download all the individual country files at ADM2 and try to merge them, will there be overlap with the national boundaries?

**Comment by DanRunfola** [comment]
If you're referring to the CGAZ ADM2, then - yes, you will see extensive simplification.  The individual polygons will have much higher vertex counts, but will overlap in many cases.
